### ISSUE TITLE
For some reason, ONLY Taulov respawns at the old arena spawn when killed in arena. (Taulov)

### ISSUE THREAD

**Issue by CreatingWithKass** [PirateCraft/dev#128]
**Describe the bug** *(A clear and concise description of what the bug is)*:
Godsy re-set the player spawn to a safe spot at arena, so players could quickly jump back into the battle. However, for ONLY Taulov, he is auto-respawned at the old player spawn, which is not safe and leads to him repeat dying if he can't TP out. 

**To Reproduce** *(Steps to reproduce the behavior)*:
Kill Taulov at arena. 

**Time and Date** *(If applicable, add the time (in GMT+1) and date (DD.MM.YYYY) of the issue)*:
05/03/2021

**Screenshots/Videos (Delete if not needed)** *(If applicable, add screenshots/videos to help explain your problem)*:
https://youtu.be/3c7PynPPLD0

**Additional context (Delete if this isn't needed)** *(Add any other context about the problem here)*:
Taulov ONLY has commodore as his rank; not quartermaster, firstmate, etc. This **could** have something to do with the odd re-spawning. I have not been able to test it with other moderators yet.

**Comment by TomLewis** [comment]
yeah this was super confusing when I setup the respawn and he was the one to test it, Legit no idea.
